components/groups/read/group-list-view/sorting.tsx

```diff
@@ -0,0 +1,113 @@
+import { memo, useCallback } from 'react'
+
```

Comment:
why do we have the same component 2 times ?  It would be better if it was a generic component. And that we could send the options as properties, so we can reuse it for other things

Reply:
done